python/vmaf/script/run_vmaf.py

```diff
@@ -32,10 +36,12 @@ def print_usage():
 
 
 def main():
+    # 检查参数数量
```

Comment:
Would prefer not to have non-English comments.

Reply:
> Would prefer not to have non-English comments.

ok, thanks for your reply